frb_codegen/src/library/commands/flutter.rs

```diff
@@ -20,3 +21,9 @@ pub fn flutter_pub_add(items: &[String]) -> anyhow::Result<()> {
         *items
     )?)
 }
+
+#[allow(clippy::vec_init_then_push)]
+pub fn flutter_pub_get(path: &PathBuf) -> anyhow::Result<()> {
```

Comment:
```suggestion
pub fn flutter_pub_get(path: &Path) -> anyhow::Result<()> {
```

(I am surprised CI does not warn this...)